packages/extension-vscode/package.json

```diff
@@ -82,6 +85,11 @@
   "license": "Apache-2.0",
   "main": "./dist/bundle/extension",
   "name": "vscode-webhint",
+  "nyc": {
+    "branches": 75,
```

Comment:
Hmm... shouldn't we add the clause to extend from our root `.nycrc` then?🤔

```json
  "nyc": {
    "extends": "../../.nycrc"
  },
```

I think `extension-browser` is missing extending from this too. If that causes them to fail, though, I'd recommend filing as an issue for a follow-up PR rather than fixing it here.

Reply:
I don't remember if I tried that, but I will do it, the thing is that I needed to fix it here to pass the build.
I can do another PR only for that, but I will need it to be merged ASAP.

Reply:
I'm fine with the change being in this PR if that's easier for you. Just wasn't sure if extending from the root `.nycrc` would introduce other issues...

Reply:
Looks like only extending is working too.